nevergrad/optimization/callbacks.py

```diff
@@ -135,3 +135,35 @@ def load_flattened(self, max_list_elements: int = 24) -> List[Dict[str, Any]]:
                         break
                     flat_data[-1][key + "#" + "_".join(str(i) for i in indices)] = value
         return flat_data
+
+    def to_hiplot_experiment(self, max_list_elements: int = 24) -> tp.Any:  # no typing here seems Hiplot is not a hard requirement
+        """Converts the logs into an hiplot experiment for display.
+
+
+        Example
+        -------
+        exp = logs.to_hiplot_experiment()
+        exp.display(force_full_width=True)
+
+        Note
+        ----
+        - You can easily change the axes of the XY plot:
+          exp.display_data(hip.Displays.XY).update({'axis_x': '0#0', 'axis_y': '0#1'})
+        - For more context about hiplot, check:
+          - blogpost: XXX
+          - github repo: https://github.com/facebookresearch/hiplot
+          - documentation: https://facebookresearch.github.io/hiplot/
+        """
+        import hiplot as hip
+        exp = hip.Experiment()
+        for xp in self.load_flattened(max_list_elements=max_list_elements):
+            dp = hip.Datapoint(
+                from_uid=xp["#parents_uids#0"] if "#parents_uids#0" in xp else None,
+                uid=xp["#uid"],
+                values={x: y for x, y in xp.items() if not (x.startswith("#") and ("uid" in x or "ask" in x))}
+            )
+            exp.datapoints.append(dp)
+        exp.display_data(hip.Displays.XY).update({'axis_x': '#num-tell', 'axis_y': '#loss'})
+        # for the record, some more options:
+        exp.display_data(hip.Displays.XY).update({'lines_thickness': 1.0, 'lines_opacity': 1.0})
```

Comment:
not sure if you really need that :)

Reply:
No, but that serves as documentation to me :D